src/test/java/com/example/java/test/junior/developer/question/QuestionControllerTest.java

```diff
@@ -0,0 +1,112 @@
+package com.example.java.test.junior.developer.question;
```

Comment:
Package not correct. You should keep tests structure THE SAME as project structure. 
So tests should be not in folders like "language", "question" etc - but in packages "controller", "service" etc.